### DISCUSSION TITLE
If I git pull will I have to remake the training weights?

### DISCUSSION THREAD

**Discussion by nazthelizard122** [ggml-org/llama.cpp#220] [Q&A] [answered]
I downloaded this project a while ago since it is the best interface with LLaMa at the moment, considering the weights are a few hundred GB total, it would be quite inconvenient to have to re-quantisize them every time I git pull, Is this an issue I should be concerned about?

**Comment by j-f1** [comment, marked as answer]
You can check the change history for `quantize.cpp` and `convert-pth-to-ggml.py` — unless those are changed, you probably don’t need to recreate the ggml files. The most recent change to those files is optional to re-apply but it improves output quality for non-Latin scripts.

**Comment by nazthelizard122** [comment]
Thanks!
